Sireesha/Practice_Java/src/Arrays/SortZerosOnesTwosArr.java

```diff
@@ -0,0 +1,33 @@
+package Arrays;
+import java.util.*;
+public class SortZerosOnesTwosArr {
+    public void sort(int arr[]) {
+    	int len=arr.length;
+    	int ptr1=0,ptr2=0;
+    	for(int i=0;i<len;i++) {
+    		if(arr[i]==0)
+    			ptr1++;
+    		else if(arr[i]==1)
+    			ptr2++;
```

Comment:
The intention of this question is to derive logic, while doing these kind of programs, 99% we need to maintain Order as well. So Just get count of 1s and 2s and 0s is not the logic and in this case order of elements will missed. 
Note; Order of elements in the sense, even though values are equal we should give first element in the first place